scripts/swift_build.sh

```diff
@@ -2,5 +2,5 @@
 
 mkdir $DEPLOY_DIRECTORY
 
-swift test -c debug --enable-test-discovery --enable-code-coverage
-cp $(swift test --show-codecov-path) deploy/codecov.json
+# Run swift test, grabbing the last line of the output (the code coverage path) then copying it to the file below.
+cp $(swift test --enable-code-coverage --show-codecov-path | tail -n1) deploy/codecov.json
```

Comment:
Walking through these changes:
- Running this command twice is redundant I think
- ` -c debug` is unnecessary, debug is the default
- The second version of this command failed since ` --enable-code-coverage` is required when using `--show-codecov-path`
- ` | tail -n1` grabs the last line of the test output which is the code coverage report that we can copy to the deploy folder